packages/main/src/i18n/messagebundle.properties

```diff
@@ -139,6 +139,18 @@ COLORPICKER_GREEN=Green
 #XTOL: Blue color for the ColorPicker control
 COLORPICKER_BLUE=Blue
 
+#XTOL: Hue value for the ColorPicker control
+COLORPICKER_HUE=Hue
+
+#XTOL: Saturation value for the ColorPicker control
+COLORPICKER_SATURATION=Saturation
+
+#XTOL: Light value for the ColorPicker control
```

Comment:
#XFLD corresponds to a label. And the text could be something like: "Label text for the ColorPicker Light input field."